hips/utils/tests/test_healpix.py

```diff
@@ -22,3 +24,17 @@ def test_boundaries():
     radec_precomp = [[264.375,  258.75,  264.375,  270.],
                      [-24.624318,  -30.,  -35.685335,  -30.]]
     assert_allclose([radec.ra, radec.dec], radec_precomp)
+
+def test_compute_healpix_pixel_indices():
+    order = 3
+    nside = hp.order2nside(order)
+
+    skycoord = SkyCoord(10, 20, unit="deg")
+    wcs_geometry = WCSGeometry.create(skydir=skycoord, shape=(10, 20), coordsys='CEL', projection='AIT', cdelt=1.0, crpix=(1., 1.))
+
+    pixels_precomp = [84, 111, 112, 113]
+    len_precomp = 39
+    pixels = compute_healpix_pixel_indices(wcs_geometry, nside)
+    print(pixels)
```

Comment:
Remove print here. Do this always before pushing to Github. There should never be print statements in library code like hips or in the tests.